### PR TITLE
Implement --repo and --policy cli flags to filter down Allstar enforcement scope

### PR DESCRIPTION
**Pull request by Gby56**
Quite a useful feature imo, especially when you want to test Allstar locally

### CODE REVIEW

File: cmd/allstar/main.go [hunk 1]
```diff
@@ -41,12 +43,45 @@ func main() {
...
+	supportedPoliciesMap := map[string]string{}
+	var supportedPoliciesMsg = ""
 
+	for i, p := range supportedPolicies {
+		var policyName = p.Name()
+		supportedPoliciesMap[policyName] = policyName
+		if i < len(supportedPolicies)-1 {
+			supportedPoliciesMsg += policyName + ", "
+		}
+		if i == len(supportedPolicies)-1 {
```
Comment (coheigea) [inline cmd/allstar/main.go]: Can we just use else here?

**Review by coheigea — CHANGES_REQUESTED** [review]
Great improvement for testing

File: cmd/allstar/main.go (hunk 1 above)
Comment (Gby56) [reply cmd/allstar/main.go]: Done

**Review by coheigea — DISMISSED** [review]

**Comment by Gby56** [comment]
I need to fix the tests !

**Comment by jeffmendoza** [comment]
Sorry they were not running automatically
image
I just changed the setting from middle to top here, hope that helps.

**Review by jeffmendoza — APPROVED** [review]

**Pull request merged**